src/components/10-atoms/input-text/index.js

```diff
@@ -366,57 +353,26 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
       }
       <div class="a-input-text__input-wrapper">
         <div class="a-input-text__input-elements">
-        ${
-          // On Safari, the caret (cursor) jumps to the end of the value, which
-          // is fixed with this approach.
-          // Downside: After a user manipulated the value manually, it cannot
-          // be updated anymore by javascript (safari only).
-          this._isSafari()
-            ? html`
-                <input
-                  id="${refId}"
-                  type="${type}"
-                  class="${classMap(inputClasses)}"
-                  autocomplete="off"
-                  name="${name}"
-                  value="${formattedValue}"
-                  placeholder="${placeholder}"
-                  aria-required="${required}"
-                  maxlength="${maxLength}"
-                  pattern="${pattern}"
-                  inputmode="${inputmode}"
-                  ?disabled="${disabled}"
-                  ?readonly="${readonly}"
-                  @input="${this.handleInput}"
-                  @focus="${this.handleFocus}"
-                  @blur="${this.handleBlur}"
-                  @keydown="${this.onKeyDown}"
-                  @keyup="${this.onKeyUp}"
-                />
-              `
-            : html`
-                <input
-                  id="${refId}"
-                  type="${type}"
-                  class="${classMap(inputClasses)}"
-                  autocomplete="off"
-                  name="${name}"
-                  .value="${formattedValue}"
-                  placeholder="${placeholder}"
-                  aria-required="${required}"
-                  maxlength="${maxLength}"
-                  pattern="${pattern}"
-                  inputmode="${inputmode}"
-                  ?disabled="${disabled}"
-                  ?readonly="${readonly}"
-                  @input="${this.handleInput}"
-                  @focus="${this.handleFocus}"
-                  @blur="${this.handleBlur}"
-                  @keydown="${this.onKeyDown}"
-                  @keyup="${this.onKeyUp}"
-                />
-              `
-        }
+        ${html`<input
```

Comment:
So, the hack was needed at some point. Starting from which (mobile) Safari version this is fixed now? We would at the very least expect some info on this in the Github issue that would be closed by this PR, as a comment.